charts/postgresql/values.yaml

```diff
@@ -15,7 +15,7 @@ global:
 image:
   registry: docker.io
   repository: bitnami/postgresql
-  tag: 11.5.0-debian-9-r60
+  tag: 11.11.0-debian-10-r28
```

Comment:
Can we depend on the stability of this image that was released just a couple of days ago?

Reply:
Maybe not, because now there's an image that's just a few hours old. 😂

Reply:
Postgres isn't run in-cluster in prod so this isn't a big deal.